components/x-audio/stories/redux-player.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
maybe worth adding https://github.com/storybooks/storybook-addon-console? or is this just for testing

Reply:
that plugin looks useful, I'll have a go at installing over on master